src/renderer/StoreChangeObserver.tsx

```diff
@@ -156,6 +156,11 @@ const StoreChangeObserver = () => {
     ipc.setEditWordEnabled(editWordEnabled);
   }, [selfSelection, editWordIndex]);
 
+  useEffect(() => {
+    const selectSentenceEnabled = getLengthOfRange(selfSelection) > 0;
```

Comment:
Should we grey out the button if the current sentence is already selected? Since it would be idempotent in that case

Reply:
Yep, sounds good. Added this functionality